src/Server/Player/PlayerLogic.py

```diff
@@ -72,8 +74,14 @@ def _send_updated_pos_hp(self):
 
 
     def _attack(self):
+        if self._can_attack:
+            self._can_attack = False
+            Timer.sch_fun(100, self.restore_attackaibility, ())
         return True  # TODO cooldown?
```

Comment:
Remove TODO.

Reply:
Seems like both in OrangeLogic and AppleLogic the _self._can_attack_ is already verified before its call.
So, it seems to be redundant, consider removing somehow from one of the places.

Reply:
Again, return value, why?